packages/integration-tests/src/utils.rs

```diff
@@ -181,3 +186,49 @@ pub fn get_dr_id(res: AppResponse) -> String {
     // https://github.com/sedaprotocol/seda-chain-contracts/issues/68
     res.events.last().unwrap().attributes[2].value.clone()
 }
+
+pub fn calculate_commitment(reveal: &str, salt: &str) -> String {
+    let mut hasher = Keccak256::new();
+    hasher.update(reveal.as_bytes());
+    hasher.update(salt.as_bytes());
+    let digest = hasher.finalize();
+    format!("0x{}", hex::encode(digest))
+}
+
+pub fn helper_commit_result(
+    app: &mut App,
+    proxy_contract: CwTemplateContract,
+    dr_id: String,
+    commitment: String,
+    sender: Addr,
+) -> Result<AppResponse, Error> {
+    let msg = ProxyExecuteMsg::CommitDataResult { dr_id, commitment };
+    let cosmos_msg = proxy_contract.call(msg).unwrap();
+    app.execute(sender, cosmos_msg.clone())
+}
```

Comment:
This is just a suggestion to provide an example how to avoid being forced to use Anyhow::Error even though we are not really benefiting from it.

```suggestion
) -> Result<AppResponse, String> {
    let msg = ProxyExecuteMsg::CommitDataResult { dr_id, commitment };
    let cosmos_msg = proxy_contract.call(msg).unwrap();

    app.execute(sender, cosmos_msg.clone())
        .map_err(|x| x.to_string())
}
```

We could also convert the Anyhow Error into one of our own errors. :)